src/main/java/io/ebean/enhance/ant/OfflineFileTransform.java

```diff
@@ -150,7 +153,7 @@ private void transformFile(File file) throws IOException, IllegalClassFormatExce
 
 		if (result != null) {
 			InputStreamTransform.writeBytes(result, file);
-			if(listener!=null) {
+			if(listener!=null && logLevel > 0) {
```

Comment:
Is this useful?  What is the motivation for this?  Controlling maven logging?

Reply:
The concrete problem was, that the OfflineFileTransformer flooded the build-log.

Motivation:
I use the offline-filetransformer in a self compiled version of the "plexus-compiler-eclipse"
the modified version supports annotation-processing for lombok & querybean.
It performs compilation and enhancement in one step. (and due the fact that the eclipse-compiler supports multi-threading, we could nearly half our maven build tim)